docs/content/latest/secure/authentication/password-authentication.md

```diff
@@ -160,3 +160,27 @@ When you [enable SCRAM-SHA-256 authentication](#enable-scram-sha-256-authenticat
 - All existing passwords were encrypted using the MD5 hashing algorithm.
 
 Because all existing passwords must be changed, you can manage the migration of these user and role passwords from MD5 to SCRAM-SHA-256 by maintaining rules in the `--ysql_hba_conf` setting to allow both MD5 passwords and SCRAM-SHA-256 passwords to work until all passwords have been migrated to SCRAM-SHA-256. For an example, see [Create a cluster that uses SCRAM-SHA-256 password authentication](#Create-a-cluster-that-uses-scram-sha-256-password-authentication) above. If you follow a similar approach for an existing cluster, you can enhance your cluster security, track and migrate passwords, and then remove the much weaker MD5 rules after all passwords have been updated.
+
+## Resetting user password
+
+There are cases where we have lost the password of a user and want to reset it. The `pg_hba.conf` file can be modified 
```

Comment:
The intro sentence is weak. Also, bringing in pg_hba.conf mention is unnecessary and confusing as explained here.

Reply:
I suggest rewording as follows (from the notes Bill K) had written on this topic:

In Postgres if the administrator password is lost or changed to an unknown value the pg_hba.conf can be modified to allow administrator access without a password. In Postgres this is a static file that is used to control client authentication. To reset the password for the postgres user, parameters are  modified in this configuration file, the database is restarted, and then the postgres user can login as postgres without a password, and reset the password.

The same is also true for YugabyteDB, although the implementation is slightly different. YugebyteDB has a ysql_hba.conf file similar to Postgres. However, unlike Postgres, the contents of the file are dynamically generated leveraging a flag called --ysql_hba_conf at cluster startup. The following steps will outline how the use of this flag can allow administrative access for the yugabyte user if the password is ever lost or changes to an unknown value.

---

Please use correct font for filenames; and add hyperlinks to tserver gflag section for ysql_hba_conf

Reply:
Reworded in https://github.com/yugabyte/yugabyte-db/pull/6286/commits/1345e460ce55942a16a34e0183b415e3986f1bf9